.nycrc

```diff
@@ -0,0 +1,16 @@
+{
+  "reporter": [
+    "lcov",
+    "text-summary"
+  ],
+  "check-coverage": true,
+  "statements": 82,
+  "branches": 68,
+  "functions": 83,
+  "lines": 82,
```

Comment:
is this going to fail if we don't hit these numbers?

Reply:
yes it would but they can be changed if needs be

Reply:
Do you think having limits is going to block us?

Reply:
I can remove this for now and add it in once we are closer to finishing everything?

Reply:
Sounds good.  Do you think when we do add it, we need to discuss and agree it as a whole in terms of what our standards are in HOF and the wider HO?

Reply:
yeah, I think that would be a good idea - encourage more consistency across the services 👍🏾

Reply:
I have removed this file now